packages/vite/src/node/optimizer/index.ts

```diff
@@ -533,15 +533,26 @@ export async function runOptimizeDeps(
     flatIdToExports[flatId] = exportsData
   }
 
+  const define = {
```

Comment:
Maybe we could have a comment that describes the reason here.

Reply:
done